### PR TITLE
fix #487 Add libunwind to nix configuration

### LINKED ISSUE #487: Add libunwind to nix configuration
https://github.com/scala-native/scala-native/pull/477#discussion_r95453380

### PR DESCRIPTION
**Pull request by MasseGuillaume**
the printStack result is a bit different on my computer.

### CODE REVIEW

File: unit-tests/src/main/scala/java/lang/ExceptionSuite.scala [hunk 1]
```diff
@@ -7,29 +7,20 @@ object ExceptionSuite extends tests.Suite {
...
-    ).mkString("\n")
-    // It's startsWith and not == as there could be additional stack
-    // activations before main in the stack trace. For example on linux
-    // there would be additional __libc_start_main and _start frames.
-    assert(sw.toString.startsWith(expected))
+    )
+
+    val stackTrace = sw.toString
+
+    assert(segments.forall(stackTrace.contains))
```
Comment (MasseGuillaume) [inline unit-tests/src/main/scala/java/lang/ExceptionSuite.scala]: YOLO

Comment (densh) [inline unit-tests/src/main/scala/java/lang/ExceptionSuite.scala]: As mentioned in the other PR, the test change does not look good to me.

**Comment by densh** [comment]
Ping @MasseGuillaume.

**Comment by MasseGuillaume** [comment]
see #517

**Pull request closed without merging**